src-tauri/src/lib.rs

```diff
@@ -275,6 +280,42 @@ async fn get_data_dir(
         .to_string())
 }
 
+#[tauri::command]
+async fn save_txt_files(app: tauri::AppHandle, content: Vec<String>) -> Result<(), String> {
```

Comment:
Maybe take a `HashMap<String, String>` here and use the key as a filename instead of hardcoding them as `log_<i>.txt`